packages/date-picker/src/vaadin-lit-date-picker.js

```diff
@@ -117,9 +117,10 @@ class DatePicker extends DatePickerMixin(InputControlMixin(ThemableMixin(Element
         this._setFocusElement(input);
         this.stateTarget = input;
         this.ariaTarget = input;
+
+        this.addController(new LabelledInputController(input, this._labelController));
```

Comment:
The component currently doesn't seem to dispose of previous instances of `LabelledInputController`. I wonder if that can be a problem.

Reply:
It can be a problem when replacing `input` (which is not tested, neither in `LabelledInputController` itself nor in elements using it). I could add `setInput()` method to address that scenario, although it's an edge case.

Reply:
Isn't it actually what's happening when adding a field to a confirm dialog?

Reply:
Yeah, after more careful consideration, I realize that the described scenario is currently impossible. The input controller doesn't observer for changes in its slot, so it creates an input element only once.